cmd/ocm/cluster/login/login.go

```diff
@@ -47,13 +49,8 @@ var Cmd = &cobra.Command{
 
 func init() {
 	flags := Cmd.Flags()
-	flags.StringVarP(
-		&args.user,
-		"username",
-		"u",
-		"",
-		"Username, will prompt if not provided",
-	)
+	flags.StringVarP(&args.user, "username", "u", "", "Username, will prompt if not provided")
```

Comment:
In other parts of the code when we need to specify an user we use just `user`. For example in the top level command (although that option is deprecated). Can we use `user` instead of `username` here as well?

I would also prefer if we stick to the convention of putting each line of the cals to `flags.WhaveverVar(...)` in a different line. It makes it easier to write longer help texts.